package.json

```diff
@@ -1,26 +1,37 @@
 {
   "name": "devcert-sanscache",
-  "version": "0.4.7",
+  "version": "0.4.8",
   "description": "Generate trusted local SSL/TLS certificates for local SSL development",
-  "main": "index.js",
+  "type": "module",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.js"
+      }
+    }
+  },
   "scripts": {
     "prepublish": "tsc"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/guybedford/devcert.git"
   },
+  "engines": {
+    "node": "^20.11.0 || >=22.0.0"
```

Comment:
Node.js 14+ support ES modules, so in theory this should work there I think.

Reply:
The current version of [glob](https://github.com/isaacs/node-glob/blob/main/changelog.md#110) requires Node 20, so I thought it would be reasonable to align the minimum versions with that.

I chose 20.11 instead of 20.0 since it provides almost the same APIs as 22.0. Node 21 is already out of maintenance.

But no problem to change this to 14+ if you think that would be better.

Reply:
In this case we should not be updating to this version of glob!

Reply:
Got it. The same applies to the latest version of rimraf but I don't see a problem there either.

[get-port](https://github.com/sindresorhus/get-port/tree/main) v7 requires Node 16, should we downgrade to v6 as well? I don't see relevant fixes/features in the current v7 releases.

Reply:
I downgraded all three dependencies and changed the `engines` field to what seems reasonable based on what the dependencies state.

I have to add that I didn't test this with an older Node.js version than 20, since the project we use this package in, does not support those anymore 😅